src/Sylius/Bundle/ApiBundle/Command/AddProductReview.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command;
+
+/** @experimental */
+class AddProductReview
+{
+    /**
+     * @var string
+     * @psalm-immutable
+     */
+    public $title;
+
+    /**
+     * @var int
+     * @psalm-immutable
+     */
+    public $rating;
+
+    /**
+     * @var string
+     * @psalm-immutable
+     */
+    public $comment;
+
+    /**
+     * @var string
+     * @psalm-immutable
+     */
+    public $product;
```

Comment:
I'm not sure because all fields in our other endpoints have fileds `product` (or other object fields), and Its mean add `iri`. Our generally idea is adding `iri` if you need to add certain object.

Reply:
Please, check all command in the cart :), in code we always use some suffix :)